designs/GcdUnit/testbench/run_tb.sh

```diff
@@ -0,0 +1 @@
+vcs -full64 -sverilog -timescale=1ns/1ps -debug_access+pp -lca -cflags "-std=c++11" +vc+abstract GcdUnitTb.sv ../rtl/GcdUnit-demo.v
```

Comment:
This might be our own code, but I think the "../rtl/GcdUnit-demo.v" is breaking modularity.